src/main/java/uk/ac/cam/cl/dtg/segue/api/AdminFacade.java

```diff
@@ -763,8 +753,6 @@ public Response findUsers(@Context final HttpServletRequest httpServletRequest,
             @QueryParam("id") final Long userId, @QueryParam("email") @Nullable final String email,
             @QueryParam("familyName") @Nullable final String familyName, @QueryParam("role") @Nullable final Role role,
             @QueryParam("schoolOther") @Nullable final String schoolOther,
-            @QueryParam("postcode") @Nullable final String postcode,
-            @QueryParam("postcodeRadius") @Nullable final String postcodeRadius,
```

Comment:
Feature change: Admins can no longer search for users by post code radius (but since we don't have IP geolocation data this wouldn't work anyway.)